src/common/meta/src/ddl/utils.rs

```diff
@@ -182,6 +184,75 @@ pub fn extract_region_wal_options(
     Ok(region_wal_options)
 }
 
+pub enum MultipleResults {
```

Comment:
Describe this enum in detail.

Reply:
resolved in [0bca295](https://github.com/GreptimeTeam/greptimedb/pull/5792/commits/0bca29552c3fc24d307306ff11406db1f492cc4f)